src/applications/personalization/profile-2/msw-mocks.js

```diff
@@ -405,3 +463,240 @@ export const addEmailAddressSuccess = () => {
     }),
   ];
 };
+
+export const deleteEmailAddressSuccess = [
+  rest.delete(`${prefix}/v0/profile/email_addresses`, (req, res, ctx) => {
+    return res(
+      ctx.json({
+        data: {
+          id: '',
+          type: 'async_transaction_vet360_email_transactions',
+          attributes: {
+            transactionId: '61ffa9bd-3290-4e4b-9480-d93fd6236dcf',
+            transactionStatus: 'RECEIVED',
+            type: 'AsyncTransaction::Vet360::EmailTransaction',
+            metadata: [],
+          },
+        },
+      }),
+    );
+  }),
+  rest.get(`${prefix}/v0/user`, (req, res, ctx) => {
```

Comment:
after a successful deletion, when `GET user/` is called we need to return a user object that no longer contains an email address as part of the `vet360ContactInformation`